include/jsonv/optional.hpp

```diff
@@ -32,8 +32,13 @@
 #       undef  JSONV_OPTIONAL_USE_STD
 #       define JSONV_OPTIONAL_USE_STD 1
 #   else
-#       undef  JSONV_OPTIONAL_USE_STD_EXPERIMENTAL
-#       define JSONV_OPTIONAL_USE_STD_EXPERIMENTAL 1
+#       if defined(_MSC_VER)
+#           undef  JSONV_OPTIONAL_USE_BOOST
```

Comment:
I think setting the definition in the project makes more sense. What I don't want is a library consumer to be surprised they pulled in a Boost dependency -- it's better if it fails to compile outright.